archivers/urbackup-client/files/openrc-urbackup-client.in

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+#
+
+name="urbackup_client"
+description="UrBackup Client"
+command=/usr/local/sbin/urbackupclientbackend
+pidfile="/var/run/urbackupclientbackend.pid"
+command_args="-w $pidfile -c /usr/local/etc/urbackup/urbackupclient.conf"
```

Comment:
```suggestion
command_args="-c /usr/local/etc/urbackup/urbackupclient.conf"
```
Remove the "-w $pidfile". The ${pidfile} location will be used by the supervise-daemon process instead, since the client backend utility is not running in daemon-mode.

Reply:
OK. I asked both the [FreeBSD port maintainer](mailto:freebsd@coombscloud.com) and the [support forum](https://forums.urbackup.org/t/what-does-the-w-flag-to-the-urbackupclientbackend-command-do/7757) what the `-w` flag does. Hopefully I hear something useful back. In the mean time I'll make that change locally over the weekend and see what effect, if any, it has.

If the change doesn't worsen the situation and no one gets back to me from the above. I'll commit it.